docker-compose.test.yml

```diff
@@ -64,6 +64,26 @@ services:
         networks:
             - agenta-network
 
+    agenta-web:
```

Comment:
why is this part of the pr

Reply:
Same as [this](https://github.com/Agenta-AI/agenta/pull/1705#discussion_r1624181360).